packages/mergebot/src/queries/all-open-prs-query.ts

```diff
@@ -3,18 +3,13 @@ import { client } from "../graphql-client";
 import { GetAllOpenPRsAndCardIDs, GetAllOpenPRsAndCardIDsVariables } from "./schema/GetAllOpenPRsAndCardIDs";
 import { noNullish } from "../util/util";
 
-const getAllOpenPRsAndCardIDsQuery: TypedDocumentNode<GetAllOpenPRsAndCardIDs, GetAllOpenPRsAndCardIDsVariables> = gql`
+const getAllOpenPRsQuery: TypedDocumentNode<GetAllOpenPRsAndCardIDs, GetAllOpenPRsAndCardIDsVariables> = gql`
```

Comment:
Doesn't modifying this require committing the query types?

Reply:
In theory yes, I think so. I think it works because I'm just deleting some code.

Reply:
Yeah, you should run `AUTH_TOKEN=$(gh auth token) pnpm run graphql-schema` in the mergebot dir. I didn't realize CI was not enforcing that this was clean.